### PR TITLE
fix AKS cluster handling

### PR DESCRIPTION
**Pull request by tarokkk**

### CODE REVIEW

File: cluster/aks.go [hunk 1]
```diff
@@ -49,7 +49,11 @@ func (c *AKSCluster) GetAPIEndpoint() (string, error) {
 	if c.APIEndpoint != "" {
 		return c.APIEndpoint, nil
 	}
-	c.APIEndpoint = c.azureCluster.Properties.Fqdn
+	cluster, err := c.GetAzureCluster()
+	if err != nil {
+
```
Comment (bonifaido) [inline cluster/aks.go]: Missing error handling.

**Comment by bonifaido** [comment]
Needs a rebase.

**Comment by baluchicken** [comment]
I am going to try this out, please wait until that.

**Comment by baluchicken** [comment]
This works please rebase and it is good to go

**Pull request merged**